lib/contextualize.js

```diff
@@ -38,14 +74,34 @@ module.exports = function contextualize(files) {
       }
     })
 
-    // Attach JSON and YML data to pages in the same directory
-    page.data = {}
+    // Attach JSON and YAML data to pages in the same directory
+//    page.data = {}
```

Comment:
No.
It is initialized already.

Since getting frontmatter was moved to `getFrontmatter()` in `./lib/files/page.js` now that initialization is done as soon as the file is created like this:
> ./lib/files/page.js

```
  getFrontmatter() {
    this.data = {}
    var data = frontmatter(this.input)
    ...
```